lib/workers/repository/dependency-dashboard.ts

```diff
@@ -245,6 +266,25 @@ export async function ensureDependencyDashboard(
 
   issueBody = appendRepoProblems(config, issueBody);
 
+  if (hasDeprecations) {
+    issueBody += '> ⚠ **Warning**\n> \n';
+    issueBody += 'The following dependencies are deprecated:\n\n';
+    issueBody += '| Datasource | Name | Replacement? |\n';
+    issueBody += '|------------|------|--------------|\n';
+    for (const manager of Object.keys(deprecatedPackages).sort()) {
+      const deps = deprecatedPackages[manager];
+      for (const depName of Object.keys(deps).sort()) {
+        const hasReplacement = deps[depName];
+        issueBody += `| ${manager} | \`${depName}\` | ${
+          hasReplacement
+            ? '![Available](https://img.shields.io/badge/available-green?style=flat-square)'
+            : '![Unavailable](https://img.shields.io/badge/unavailable-orange?style=flat-square)'
```

Comment:
Could we go with this maybe?

- yes, green badge
- no, red badge

Yes/no is simpler than available/unavailable. The table heading already gives the context for this column.

Reply:
This doesn't work if combined with your suggestion of "Replacement PR?". Because there might not be a PR - it might require approval. It's misleading to say Replacement PR = yes if there's no PR, but it's fine to say Replacement PR = available. I prefer "available" actually, I think Yes is simpler but in this case worse

Reply:
Let's keep available/unavailable then!